compiler-core/src/build/package_compiler.rs

```diff
@@ -304,11 +309,13 @@ where
             let name = format!("{}.cache_meta", &module_name);
             let path = artefact_dir.join(name);
             let info = CacheMetadata {
+                name: module.name.clone(),
                 mtime: module.mtime,
                 codegen_performed: self.perform_codegen,
                 dependencies: module.dependencies.clone(),
                 fingerprint: SourceFingerprint::new(&module.code),
                 line_numbers: module.ast.type_info.line_numbers.clone(),
+                interface: package_interface::ModuleInterface::from_module(module),
```

Comment:
The module interface is now always being generated. It should only be built as needed and it must not be part of the build cache.

Reply:
So the goal is to not increase the cache size, correct? If so, when `gleam export package-definitions` is ran, we'd somehow convert the `CachedModule` or `CacheMetadata` into a `ModuleInterface`, correct?

Reply:
Although to generate a `ModuleInterface` we need the AST, so we'd need to analyze (at least partially) the cached modules. So I guess it's a tradeoff between more time spent building + a bigger cache v.s. more time spent exporting.

Reply:
Makes sense. Fixed in commit 20132488dac1195825fb68fd84d4d85822e2192d